templates/ts/client-node.ts

```diff
@@ -1,8 +1,8 @@
 // tslint:disable
 import fetch from 'node-fetch';
-import { RequestInit } from 'node-fetch';
+import { RequestInit, Response } from 'node-fetch';
```

Comment:
Not reviewing this file, assuming same diff as the browser code.

Reply:
Verified by diff.